app/assets/stylesheets/layout.scss

```diff
@@ -432,6 +429,11 @@ hr {
   }
 }
 
+.desktop-invitation-content {
```

Comment:
Open question (that I do not know the answer to) - Do we need the custom CSS or could this be achieved by conditionally adding/removing Primer classes?

Reply:
We do have a primer class for auto margin but I can't find anything for 50% width so we'd still have to add this class 😞I'll leave it as-if for now since we need the class either way and it's easier to distinguish the difference between mobile vs desktop in separate classes